docs/getting-started/cli.md

```diff
@@ -100,16 +102,16 @@ semgrep ci --dry-run
 
 This can be helpful to verify the results of a specific ruleset or to see how your findings change based on the rulesets you choose for your scans.
 
-### Scan using Semgrep's OSS engine
+### Scan using OSS-only analysis (single-function)
 
-To scan your project with Semgrep's OSS Engine, even though you have the Pro Engine enabled in Semgrep Cloud Platform:
+To scan your project using exclusively open-source Semgrep, even though you have proprietary cross-file analysis enabled in Semgrep Cloud Platform:
 
 ```console
 semgrep ci --oss-only
 ```
 
 :::info
-See [Semgrep Pro versus Semgrep OSS](/semgrep-pro-vs-oss) for information on the differences between Semgrep Pro and OSS Engines.
+See [Semgrep Pro versus Semgrep OSS](/semgrep-pro-vs-oss) for information on the differences between Semgrep's proprietary analyses and its open-source analysis.
```

Comment:
```suggestion
See [Semgrep Pro versus Semgrep OSS](/semgrep-pro-vs-oss) for information on the differences between Semgrep's proprietary analyses and its open-source analyses.
```

Not a big thing, but I think we should be consistent in our usage of singular vs. plural